packages/core/admin/admin/src/pages/UsecasePage/index.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useIntl } from 'react-intl';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { pxToRem, useNotification } from '@strapi/helper-plugin';
+import { Main } from '@strapi/design-system/Main';
+import { Flex } from '@strapi/design-system/Flex';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+import { Typography } from '@strapi/design-system/Typography';
+import { Select, Option } from '@strapi/design-system/Select';
+import { TextInput } from '@strapi/design-system/TextInput';
+import { TextButton } from '@strapi/design-system/TextButton';
+import { Button } from '@strapi/design-system/Button';
+import Logo from '../../components/UnauthenticatedLogo';
+import UnauthenticatedLayout, { LayoutContent } from '../../layouts/UnauthenticatedLayout';
+
+export const options = [
+  { label: 'Front-end developer', value: 'front_end_developer' },
+  { label: 'Back-end developer', value: 'back_end_developer' },
+  { label: 'Full-stack developer', value: 'full_stack_developer' },
+  { label: 'Content Manager', value: 'content_manager' },
+  { label: 'Content Creator', value: 'content_creator' },
+  { label: 'Other', value: 'other' },
```

Comment:
Shouldn't the values be translated to the languages we support in the login interface?

Reply:
indeed I should create an InltLabel, good catch, thanks!